composer.json

```diff
@@ -5,7 +5,7 @@
     "homepage": "https://airbrake.io",
     "license": "MIT",
     "require": {
-        "php": ">=5.4"
+        "php": "^5.4.7 || ^7.0"
```

Comment:
Being bad doesn't mean we shouldn't support it :) Ideally, we want to support as many versions as we can, so I would drop versions with a greater care.